src/main/java/oshi/hardware/stores/HWNetworkStore.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2016 com.github.dblock.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * enrico[dot]bianchi[at]gmail[dot]com
+ *    com.github.dblock - initial API and implementation and/or initial documentation
+ */
+package oshi.hardware.stores;
+
+import javax.json.Json;
+import javax.json.JsonBuilderFactory;
+import javax.json.JsonObject;
+import oshi.json.NullAwareJsonObjectBuilder;
+import oshi.json.OshiJsonObject;
+
+/**
+ * Store object of network interfaces attributes
+ *
+ * @author enrico[dot]bianchi[at]gmail[dot]com
+ */
+public class HWNetworkStore implements OshiJsonObject {
+
+    private JsonBuilderFactory jsonFactory = Json.createBuilderFactory(null);
+
+    private String name;
+    private String macaddr;
+    private String ipaddr;
+    private String ipv6addr;
+    private long bytesRecv;
+    private long bytesSent;
+    private long packetsRecv;
+    private long packetsSent;
+
```

Comment:
Let's add, as a minimum, MTU (maximum transmission unit) and speed/baudrate (Bits per second) to these values as I believe they're available on all 3 platforms.

For reference to see if there are any other stats you want (dropped packets, errors, etc.) compare the linux available data with the data available on OS X in [if_data64](http://fxr.watson.org/fxr/source/bsd/net/if_var.h?v=xnu-792.6.70#L178) and on Windows in [MIB_IFROW](https://msdn.microsoft.com/en-us/library/windows/desktop/aa366836(v=vs.85).aspx)